returns/contrib/hypothesis/laws.py

```diff
@@ -53,9 +79,10 @@ def check_all_laws(
         - https://mmhaskell.com/blog/2017/3/13/obey-the-type-laws
 
     """
-    settings = _Settings(
+    settings = Settings(
```

Comment:
Maybe we should assert that users cannot pass both `use_init` and `container_stratregy` 🤔 ?

Reply:
I've added a `__post_init__` check for `Settings`. There are already overloads for `check_all_laws`, so that function should be safe. Ideally, we would make the inputs mutually exclusive in the types, but I don't know how to do that in a backward-compatible way. Leaving it at this.